### ISSUE TITLE
Translations not included for classes without text symbols

### ISSUE THREAD

**Issue by kjetil-kilhavn** [abapGit/abapGit#5377]
As far as I can tell the table D010TINF only has entries for objects with text elements. Text symbols are the only text elements for classes, so a class without any text symbols will not have an entry in D010TINF.

The support for multiple languages in serialize_xml() relies on entries in D010INF which means that for classes without text symbols only the main language documentation will be serialized.

Class zcl_abapgit_object_clas,method serialize_xml:
```
    " Table d010tinf stores info. on languages in which program is maintained
    " Select all active translations of program texts
    " Skip main language - it was already serialized
    SELECT DISTINCT language
      INTO TABLE lt_langu_additional
      FROM d010tinf
      WHERE r3state  = 'A'
        AND prog     = mv_classpool_name
        AND language <> mv_language.
```

**Comment by kjetil-kilhavn** [comment]
Not sure if this is worth fixing - perhaps the effort is better spent extending the LXE approach support to classes?

**Comment by mbtools** [comment]
I will add the fix to #5374

**Comment by kjetil-kilhavn** [comment]
That's great. I discovered this when looking into #5373. As you wrote @mbtools this is far from a trivial fix, as there can be examples of classes where the languages to which the short texts, long texts and text elements have been translated differ significantly. So I started created an example where I translated some of the documentation, and then when I debugged I discovered it wasn't even *trying* to read the translated texts :-)
